Sources/ZKSync/Transport/Receipt/PollingTransactionReceiptProcessor.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PollingTransactionReceiptProcessor.swift
+//  ZKSync
+//
+//  Created by Maxim Makhun on 7/14/21.
+//
+
+import Foundation
+import PromiseKit
+
+/// Helper class, which allows to wait until transaction is completed.
+public class PollingTransactionReceiptProcessor {
+    
+    /// Time interval between polls.
+    public let pollInterval: DispatchTimeInterval
+    
+    /// Amount of attempts, which will be used by `PollingTransactionReceiptProcessor` to verify whether
+    /// transaction was commited or not.
+    public let attempts: Int
+    
+    let provider: Provider
+    
+    let semaphore = DispatchSemaphore(value: 0)
+    
+    /// Initializer, which allows to create `PollingTransactionReceiptProcessor` instance, based on `Provider` instance.
+    /// - Parameters:
+    ///   - provider: Provider instance.
+    ///   - pollInterval: Time interval between polls.
+    ///   - attempts: Amount of attempts.
+    public init(_ provider: Provider, pollInterval: DispatchTimeInterval, attempts: Int) {
```

Comment:
We should have some kind of the default value here so that the users don't have to think about these values too much

Reply:
Added default values, which are similar to implementation in `zksync-java`.